### PR TITLE
Update application.properties to remove overly-verbose logging levels

### PR DESCRIPTION
**Pull request by JohnKallies**

### CODE REVIEW

File: api/src/main/resources/config/application-dev.properties [hunk 1]
```diff
@@ -5,14 +5,15 @@ app.baseUrl=${BASE_URL:http://localhost:8080}
 
 # Spring Boot will serve frontend files via port 8080
 # If you are using ng serve to proxy static files built by Angular, set OSMT_FRONT_END_PORT to 4200
-#app.frontEndPort=${OSMT_FRONT_END_PORT:4200}
```
Comment (JohnKallies) [inline api/src/main/resources/config/application-dev.properties:8]: This should have never been committed. Either provide the env var, or use 8080

File: api/src/main/resources/config/application.properties [hunk 2]
```diff
@@ -1,12 +1,3 @@
-#logging.level.org.springframework=DEBUG
```
Comment (JohnKallies) [inline api/src/main/resources/config/application.properties:1]: Moved to dev, and pruned out some old news from more than a year ago.

File: api/src/main/resources/config/application.properties [hunk 3]
```diff
@@ -1,12 +1,3 @@
...
-
-# This property was added to suppress ElasticSearch RestClient warnings after upgrading
-# ElasticSearch to 7.16.2 for log4j vulnerabilities. We would expect this property to be
-# removed after upgrading to spring-data-elasticsearch 4.2.x
-# https://docs.spring.io/spring-data/elasticsearch/docs/current/reference/html/#preface.requirements
-logging.level.org.elasticsearch.client.RestClient=DEBUG
```
Comment (jchavez137) [inline api/src/main/resources/config/application.properties:7]: I think we could keep this line disabled instead.

**Review by jchavez137 — APPROVED** [review]

File: api/src/main/resources/config/application.properties (hunk 3 above)
Comment (jchavez137) [reply api/src/main/resources/config/application.properties:7]: oops prod. ok!

Comment (JohnKallies) [reply api/src/main/resources/config/application.properties:7]: It is moved to the application-dev.properties file, and deleted from here.

**Pull request merged**